src/status_im/chat/subs.cljs

```diff
@@ -75,8 +76,11 @@
  :<- [:get :keyboard-height]
  (fn [kb-height]
    (cond
-     (and platform/iphone-x? (> kb-height 0)) (- kb-height 34)
-     platform/ios? kb-height
+     (and platform/iphone-x? (> kb-height 0))
```

Comment:
what do you think about managing this outside of the event loop? doesn't need to be done in that PR

Reply:
does it cause any issues currently?